tests/utils/coordinates.test.js

```diff
@@ -24,7 +24,7 @@ suite('utils.coordinates', function () {
     });
 
     test('parses null', function () {
-      assert.equal(coordinates.parse(null), null);
+      assert.equal(coordinates.parse(null), undefined);
```

Comment:
Then we need to update `parse` impl, not the test code.

https://github.com/aframevr/aframe/blob/master/src/utils/coordinates.js#L28-L30

Probably we first need to make clear the specification.